utils/docker/Dockerfile.centos.8

```diff
@@ -22,7 +22,7 @@ ARG CB0
 # package must be installed and the GPG key it provides must be
 # installed.
 ARG REPO_URL
-ARG REPO_DISTRO
+ARG REPO_DISTRO_X
```

Comment:
What's this renaming all about?

Reply:
Jenkins is giving the wrong value for REPO_DISTRO still, it's giving a centos 7 url so this change is to temporarily disable this code-block until nexus can get a rocky repo and Jenkins/pipeline-lib can be changed to pass the right url for it.  Until then all packages will come from upstream, however as it's docker this won't be happening very often.

Reply:
> until nexus can get a rocky repo

It already has rocky repos.  However,

> Jenkins/pipeline-lib can be changed to pass the right url for it.

Why can't that change be made to pass the CentOS 8 reference instead of CentOS 7?

Reply:
Good to hear it's already got nexus.  Yes that change could be made but I'm not in possession of the information that would enable me to make that change.

Reply:
What information do you need?

Repos for Rocky have the same format as CentOS where you `s/centos/rocky/g`.

But still, that change shouldn't even be necessary.  We still have CentOS 8 in our repos and it should "just work".  But maybe I see what your problem is.  Currently our default O/S is EL7.  So unless your stage name or an explicit target argument tells CI otherwise, it will use EL7.  That is what is making REPO_DISTRO use CentOS 7 and not CentOS 8.

This is evidenced by the **Could not determine target in Fault injection testing, defaulting to EL7** message in your stage steps:

![image](https://user-images.githubusercontent.com/559317/154694932-24d38d65-22ba-441c-80a9-01ffa875faa2.png)

Your solutions here are to either:

1. extend [dockerBuildArgs()](https://github.com/daos-stack/pipeline-lib/blob/master/vars/dockerBuildArgs.groovy) to take a `target` argument and then add that parameter to your `dockerBuildArgs()` arguments as `centos8` or
2. Add **CentOS 8** to your stage name in `Jenkinsfile`.

Reply:
Would you rather I revert this back to CentOS 7, wait until you've made the pipeline-lib changes or land as is?

Reply:
> Do we really want to migrate testing to an obsolete distro?

That is literally in the process of being moved to Rocky Linux.
 
> I'm really just trying to add the suppression file changes here so we don't experience so many test failures.

Sure, so do whatever you want.  I just don't want to land Dockerfile changes that are going to break that imminent migration and add to it's effort.  So if it's easiest for you to leave it on EL7, moving it will just have to wait until the Rocky work is done and a task to move all other testing from EL7 to EL8 gets priority.